src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Red Hat OpenShift
+description: How to install Okteto onto Red Hat OpenShift
+sidebar_label: Red Hat OpenShift
+id: openshift
+pagination_prev: get-started/install/index
+pagination_next: get-started/install-okteto-cli
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+
+This guide will walk you through the process of installing Okteto in Red Hat OpenShift.
```

Comment:
From a strategy perspective, we don't expect people to self-install Okteto. I think it would be valuable to include a call to action that reflects this. Sometimes that encourages people to "talk to us"

Reply:
Added a link to get into a demo call, do we have anything better?